.rubocop.yml

```diff
@@ -1,12 +1,34 @@
 AllCops:
-  TargetRubyVersion: '2.2.0'
+  TargetRubyVersion: '2.7.0'
 
+# Naming section allows you to enforce consistent naming conventions across your codebase.
 Naming/FileName:
   Exclude:
     - 'lib/omniauth-mlh.rb'
 
+# Metrics section allows you to enforce code complexity rules to keep the codebase maintainable.
 Metrics/BlockLength:
   Exclude:
     - 'Rakefile'
     - '**/*.rake'
     - 'spec/**/*'
+
+# Two spaces, no tabs (for indentation).
+Layout/IndentationWidth:
+  Enabled: true
+
+# No trailing whitespace.
+Layout/TrailingWhitespace:
+  Enabled: true
+
+# Blank lines should not have any spaces.
+Layout/TrailingEmptyLines:
+  Enabled: true
+
+# Use Ruby >= 1.9 syntax for hashes. Prefer { a: :b } over { :a => :b }.
+Style/HashSyntax:
```

Comment:
This is enabled by default: https://github.com/rubocop/rubocop/blob/master/config/default.yml#L3912